roles/network_plugin/flannel/templates/cni-flannel.yml.j2

```diff
@@ -30,9 +30,11 @@ data:
     }
   net-conf.json: |
     {
+{%   if not enable_ipv6only_stack_networks %}
       "Network": "{{ kube_pods_subnet }}",
       "EnableIPv4": true,
-{%   if enable_dual_stack_networks %}
+{%   endif %}
+{%   if enable_dual_stack_networks or enable_ipv6only_stack_networks %}
       "EnableIPv6": true,
```

Comment:
Same here, since you're modifying the template take the opportunity to fix the spacing.